### ISSUE TITLE
Bodymovin not showing previews.  Lottie files come out blank after export

### ISSUE THREAD

**Issue by ccr-515** [airbnb/lottie-web#2799]
![](https://user-images.githubusercontent.com/84052709/172921991-09d49d82-ba54-46d8-9d5f-651f148bb1fd.png)

**Tell us about your environment**

* **Browser and Browser Version:** Chrome 102
* **After Effects Version:** 22.4

**What did you do? Please explain the steps you took before you encountered the problem.**

I created a series of images to use for a header. When I try to render it from Bodymovin first it shows up blank in the preview mode (image attached) and then it shows up completely blank in the rendered .JSON

**What did you expect to happen?**
I was expecting a lottie animation to play in the preview or at the very least to be able to use it once it has been rendered.

**What actually happened? Please include as much _relevant_ detail as possible.**
When I go to preview the animation it is completely blank. After uploading the .JSON to webflow it also shows up completely blank.

**Please provide a download link to the After Effects file that demonstrates the problem.**

https://www.dropbox.com/s/kjuaghzpnakiqxw/about%20me%20header.aep?dl=0

Sorry for the double post!

**Comment by bodymovin** [comment]
I've answered this issue here
https://github.com/airbnb/lottie-web/issues/2796

**Comment by ccr-515** [comment]
> 

This is a new issue. Sorry! Please check dropbox link

**Comment by bodymovin** [comment]
How are you testing the animation? Can you share a link with the animation running?

**Comment by ccr-515** [comment]
> How are you testing the animation? Can you share a link with the animation running?

Yes this is the link to see the animation in mp4. https://www.dropbox.com/s/nbw03he2jy75h5d/about%20me%20header.mp4?dl=0
